cardano-node/src/Cardano/CLI/Tx/Generation.hs

```diff
@@ -854,7 +809,7 @@ createMoreFundCoins llTracer
                                              txOut
                                              []
   liftIO $ forM_ splittingTxs $ \(tx, txDetailsList) -> do
-    submitTx nCli pInfoConfig (node topologyInfo) tx llTracer
```

Comment:
Wouldn't it make sense to supply a `NodeId` on the command line and pipe it here rather than always submit the transaction to the same node?

Reply:
> always submit the transaction to the same node?

This is about genesis transaction and splitting transactions only. Main transactions will be send to multiple nodes (if user defined more than one `--target-node`).

Moreover, `--node-id` CLI parameter is already removed. Later we'll change this code: probably `submitTx` won't be used at all.